Ginger/Ginger/UserControlsLib/TextEditor/ValueExpression/ValueExpressionEditor.cs

```diff
@@ -130,15 +130,19 @@ void AddVars(List<ICompletionData> list)
             {
                 list.Add(GETVariableTCD(v));
             }
-
-            foreach (VariableBase v in mContext.BusinessFlow.Variables)
+            if (mContext.BusinessFlow != null)
             {
-                list.Add(GETVariableTCD(v));
+                foreach (VariableBase v in mContext.BusinessFlow.Variables)
+                {
+                    list.Add(GETVariableTCD(v));
+                }
             }
-
-            foreach (VariableBase v in mContext.BusinessFlow.CurrentActivity.Variables)
+            if (mContext.BusinessFlow.CurrentActivity != null)
```

Comment:
For this condition you should check both, mContext.BusinessFlow!=null  && mContext.BusinessFlow.CurrentActivity!=null